buildbot-host/buildmaster/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV BASEDIR=/var/lib/buildbot/masters/default
 RUN mkdir -p $BASEDIR
 
 # Required for latent docker buildslaves
-RUN . /buildbot_venv/bin/activate && pip install docker==5.0.3 || false
+RUN . /buildbot_venv/bin/activate && pip install treq docker==5.0.3 || false
```

Comment:
Please try to remove the docker version pinning together with the 3.7.0 update. I think it should be fixed in the new version.

Reply:
tried but this is not fixed yet. leaving as it is

Reply:
okay, we can look at it another time